resources/common.go

```diff
@@ -22,15 +22,15 @@ import (
 //Resource is an interface for AppController supported resources
 type Resource interface {
 	Key() string
-	Status() (string, error)
+	Status(meta map[string]string) (string, error)
 	Create() error
 	UpdateMeta(map[string]string) error
```

Comment:
If will be passing `meta` into status - IMO there is no reason to keep this method.